persistence-worker/main.py

```
@@ -47,34 +67,101 @@ async def save_to_database(self, pool):
+                            STREAM_KEY,
+                            CONSUMER_GROUP,
+                            CONSUMER_NAME,
+                            min_idle_time=0,
+                            message_ids=[msg_id],
+                        )
+                        for entry_id, fields in claimed:
+                            await self._process_entry(entry_id, fields)
+
+                # 버퍼에 있는 것들 저장
+                await self.save_to_database(pool, redis_client)
+                print(f"[PENDING] Processed pending messages")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)